<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-KNODTNC" name="Regression" tests="39" failures="6" timestamp="21 Aug 2020 09:44:42 GMT" time="0.368" errors="0">
  <testcase name="qtp" time="0.01" classname="PracticesBYTestNG.TestNGKeyWoards"/>
  <testcase name="qtp" time="0.002" classname="PracticesBYTestNG.TestNGKeyWoards"/>
  <testcase name="qtp" time="0.002" classname="PracticesBYTestNG.TestNGKeyWoards"/>
  <testcase name="qtp" time="0.005" classname="PracticesBYTestNG.TestNGKeyWoards"/>
  <testcase name="qtp" time="0.002" classname="PracticesBYTestNG.TestNGKeyWoards"/>
  <testcase name="qtp" time="0.002" classname="PracticesBYTestNG.TestNGKeyWoards"/>
  <testcase name="qtp" time="0.002" classname="PracticesBYTestNG.TestNGKeyWoards"/>
  <testcase name="qtp" time="0.002" classname="PracticesBYTestNG.TestNGKeyWoards"/>
  <testcase name="qtp" time="0.006" classname="PracticesBYTestNG.TestNGKeyWoards"/>
  <testcase name="qtp" time="0.001" classname="PracticesBYTestNG.TestNGKeyWoards"/>
  <testcase name="testing" time="0.006" classname="PracticesBYTestNG.TestNGKeyWoards">
    <failure type="java.lang.ArithmeticException" message="/ by zero">
      <![CDATA[java.lang.ArithmeticException: / by zero
	at PracticesBYTestNG.TestNGKeyWoards.testing(TestNGKeyWoards.java:39)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- testing -->
  <testcase name="rctest" time="0.014" classname="PracticesBYTestNG.TestNGKeyWoards">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
	at PracticesBYTestNG.TestNGKeyWoards.rctest(TestNGKeyWoards.java:56)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- rctest -->
  <testcase name="Deve" time="0.0" classname="PracticesBYTestNG.TestNGKeyWoards">
    <skipped/>
  </testcase> <!-- Deve -->
  <testcase name="qeewrtp" time="0.002" classname="PracticesBYTestNG.ThirdTest"/>
  <testcase name="qeewrtp" time="0.003" classname="PracticesBYTestNG.ThirdTest"/>
  <testcase name="qeewrtp" time="0.001" classname="PracticesBYTestNG.ThirdTest"/>
  <testcase name="qeewrtp" time="0.002" classname="PracticesBYTestNG.ThirdTest"/>
  <testcase name="qeewrtp" time="0.004" classname="PracticesBYTestNG.ThirdTest"/>
  <testcase name="qeewrtp" time="0.001" classname="PracticesBYTestNG.ThirdTest"/>
  <testcase name="qeewrtp" time="0.002" classname="PracticesBYTestNG.ThirdTest"/>
  <testcase name="qeewrtp" time="0.003" classname="PracticesBYTestNG.ThirdTest"/>
  <testcase name="qeewrtp" time="0.005" classname="PracticesBYTestNG.ThirdTest"/>
  <testcase name="qeewrtp" time="0.001" classname="PracticesBYTestNG.ThirdTest"/>
  <testcase name="tesgdfgting" time="0.003" classname="PracticesBYTestNG.ThirdTest">
    <failure type="java.lang.ArithmeticException" message="/ by zero">
      <![CDATA[java.lang.ArithmeticException: / by zero
	at PracticesBYTestNG.ThirdTest.tesgdfgting(ThirdTest.java:26)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- tesgdfgting -->
  <testcase name="rctjhghjest" time="0.006" classname="PracticesBYTestNG.ThirdTest">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
	at PracticesBYTestNG.ThirdTest.rctjhghjest(ThirdTest.java:43)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- rctjhghjest -->
  <testcase name="Dehggfhe" time="0.0" classname="PracticesBYTestNG.ThirdTest">
    <skipped/>
  </testcase> <!-- Dehggfhe -->
  <testcase name="qdfdsftp" time="0.005" classname="PracticesBYTestNG.SecondTesting"/>
  <testcase name="qdfdsftp" time="0.002" classname="PracticesBYTestNG.SecondTesting"/>
  <testcase name="qdfdsftp" time="0.001" classname="PracticesBYTestNG.SecondTesting"/>
  <testcase name="qdfdsftp" time="0.002" classname="PracticesBYTestNG.SecondTesting"/>
  <testcase name="qdfdsftp" time="0.002" classname="PracticesBYTestNG.SecondTesting"/>
  <testcase name="qdfdsftp" time="0.001" classname="PracticesBYTestNG.SecondTesting"/>
  <testcase name="qdfdsftp" time="0.001" classname="PracticesBYTestNG.SecondTesting"/>
  <testcase name="qdfdsftp" time="0.001" classname="PracticesBYTestNG.SecondTesting"/>
  <testcase name="qdfdsftp" time="0.002" classname="PracticesBYTestNG.SecondTesting"/>
  <testcase name="qdfdsftp" time="0.001" classname="PracticesBYTestNG.SecondTesting"/>
  <testcase name="testidsfdng" time="0.001" classname="PracticesBYTestNG.SecondTesting">
    <failure type="java.lang.ArithmeticException" message="/ by zero">
      <![CDATA[java.lang.ArithmeticException: / by zero
	at PracticesBYTestNG.SecondTesting.testidsfdng(SecondTesting.java:26)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- testidsfdng -->
  <testcase name="rctdfdfest" time="0.002" classname="PracticesBYTestNG.SecondTesting">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
	at PracticesBYTestNG.SecondTesting.rctdfdfest(SecondTesting.java:43)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- rctdfdfest -->
  <testcase name="Dedfdsfve" time="0.0" classname="PracticesBYTestNG.SecondTesting">
    <skipped/>
  </testcase> <!-- Dedfdsfve -->
</testsuite> <!-- Regression -->
